browser/js/app-initializer.js

```diff
@@ -104,7 +104,13 @@ export class AppInitializer {
 				.register(flags)
 				.catch(() => { });
 
-			serviceWorker.message({ type: 'updateCache', data: {}});
+			serviceWorker
+				.message({ type: 'updateCache', data: {}})
+				.catch((err) => {
+					if (err.toLowerCase() !== 'service worker unavailable') {
+						return Promise.reject(err);
```

Comment:
simpler to just `throw err`?

Reply:
Ha... we had this discussion a few minutes ago. I just decided to do this since that's how it was being "thrown" in `n-service-worker`. Happy to change to `throw err` if it's preferred though. 👍